readme.md

```diff
@@ -0,0 +1,45 @@
+# WP Factories
+
```

Comment:
You can add a badge to follow the version here : 

```md
[![Packagist Version](https://img.shields.io/packagist/v/studiometa/wp-factories?style=flat-square)](https://packagist.org/packages/studiometa/wp-factories)
```

[![Packagist Version](https://img.shields.io/packagist/v/studiometa/wp-factories?style=flat-square)](https://packagist.org/packages/studiometa/wp-factories)